### PR TITLE
add the documentation for the ip_tags

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Add the documentation for the azure_rm_publicipaddress's ip_tags

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_publicipaddress.py
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
@yuwzho I have added the documentation for the azure_rm_publicipaddress's ip_tags. Please review! Thanks!

File: lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py [hunk 1]
```diff
@@ -75,6 +75,11 @@
         description:
             - List of IpTag associated with the public IP address.
             - Each element should contain type:value pair.
+        suboptions:
+            type: 
+                description: Sets the ip_tags type: Example FirstPartyUsage.
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py]: This is an invalid yaml.

We usually write the description as a list. BTW, there is no need to add the example. if there is ":" in the sentence, the whole line should be quoted.

```yml
description:
    - "XXXXX"
```

**Review by yuwzho — CHANGES_REQUESTED** [review]

**Comment by Fred-sun** [comment]
@yuwzho I submitted the new changed, but the shippable check is wrong, without any hint? Could you please help to check the problem? Thanks!

File: lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py [hunk 2]
```diff
@@ -75,6 +75,11 @@
         description:
             - List of IpTag associated with the public IP address.
             - Each element should contain type:value pair.
+        suboptions:
+            type:
```
Comment (tao12345666333) [inline lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py]: thanks.

but there has a lint error `trailing spaces`.

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

**Comment by tao12345666333** [comment]
After CI goes green you need squash your commits.

**Comment by zikalino** [comment]
@tao12345666333 no need to squash now, will be squashed while merging the pr.

**Comment by Fred-sun** [comment]
ready_for_review

**Comment by yuwzho** [comment]
shipit

**Review by yuwzho — APPROVED** [review]

**Comment by tao12345666333** [comment]
> @tao12345666333 no need to squash now, will be squashed while merging the pr.

Thank you for reminding. :+1: @zikalino

**Pull request merged**